test/Driver/batch_mode_remark.swift

```diff
@@ -0,0 +1,12 @@
+// Ensure that driver issues a remark iff in batch mode.
+//
+// RUN: %swiftc_driver -whole-module-optimization -enable-batch-mode   %S/../Inputs/empty.swift  2>&1 | %FileCheck %s
+// RUN: %swiftc_driver -whole-module-optimization -enable-batch-mode   %S/../Inputs/empty.swift  2>&1 | %FileCheck %s
```

Comment:
What's the difference between this line and the previous one?

Also, would it make sense to test what no arguments at all does here?

Reply:
D'Oh! That's what I meant to do there. Fixed.